www/src/templates/component-page-template.tsx

```diff
@@ -136,7 +138,10 @@ export default function PageTemplate({
             <p className="small mb-0">{mdx.frontmatter.notes}</p>
           </Alert>
         )}
-        <h1 className="mb-4">{mdx.frontmatter.title}</h1>
+        <div className="d-flex justify-content-between">
+          <h1 className="mb-4">{mdx.frontmatter.title}</h1>
+          <LeaveFeedback as={Hyperlink} />
```

Comment:
It looks like this defaults to using `Nav.Link` styles by default, even though most uses of `LeaveFeedback` don't actually intend to use those styles. I wonder if the use of `LeaveFeedback` in the footer should be the exception that gets the `Nav.Link` styles rather than having to override that in 2 of the 3 places the `LeaveFeedback` component is used.

Is there a way to perhaps make this component a bit more flexible so the differing styles in the various use cases don't conflict?

Reply:
Thanks for the review! I replaced the top link and the bottom link with buttons, I also changed the implementation of links in the site footer. It seems to me that all links that lead to third-party resources should have rel="noopener noreferrer" and target="_blank ".